packages/core/src/tools/ToolsManager.ts

```diff
@@ -24,4 +31,17 @@ export default class ToolsManager {
         throw new Error(`Unknown tool: ${toolName}`);
     }
   }
+
+  /**
+   * Returns inline tools got from the EditorConfig tools
+   */
+  public getInlineTools(): InlineToolWithName[] {
+    return [{
+      name: createInlineToolName('Bold'),
+      tool: new BoldInlineTool(),
+    }, {
+      name: createInlineToolName('Italic'),
+      tool: new ItalicInlineTool(),
+    }];
```

Comment:
```suggestion
    return {
      bold: BoldInlineTool,
      italic: ItalicInlineTool,
    };
```

class construction should be somewhere else